scripts/update_emojis.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Copyright 2018-2022 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Update the list of emojis in `lib/streamlit/emojis.py.
+
+This script requires the emoji package to be installed: pip install emoji.
+"""
+
+import os
+import re
+
+from emoji.unicode_codes.data_dict import EMOJI_DATA  # type: ignore
+
+BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+EMOJI_SET_REGEX = re.compile(r"ALL_EMOJIS.+?\)", re.DOTALL)
+EMOJIS_SCRIPT_PATH = os.path.join(BASE_DIR, "lib", "streamlit", "emojis.py")
+TAB = "    "
+
+emoji_unicodes = set(EMOJI_DATA.keys())
+
+generated_code = f"ALL_EMOJIS = set(\n{TAB}" + "{\n"
+for emoji in sorted(emoji_unicodes):
+    generated_code += (2 * TAB) + f'"{emoji}",\n'
+generated_code += TAB + "}\n)"
```

Comment:
This probably doesn't matter too much either way (it'll probably be a few tens of KB smaller at most), but we'd save a decent amount of space if we didn't try to prettily format the `emojis.py` file and allowed the set of emoji to be a single line.

The lack of nice formatting likely won't matter given that the `emojis.py` file will be machine-generated and likely never human-read.

Reply:
Another thing that's similarly inconsequential (but may be worth fixing if it also seems worth to make this one line) is that this runs in quadratic time due to the repeated string concatenation. This definitely doesn't matter at all since it's a manually-run update script that we'll probably run once every few months at most, but I felt compelled to point this out 😆

Reply:
Yep, wanted to have it comply with black formatting. But we can deactivate formatting for this file 👍 Should be more performant now as well 😅